<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CollectorWindow</class>
 <widget class="QWidget" name="CollectorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>981</width>
    <height>1259</height>
   </rect>
  </property>
  <property name="font">
   <font/>
  </property>
  <property name="windowTitle">
   <string>光谱采集互动</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget:window {
    /* Borders around the code editor and debug window */
    background-color: #263238;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="2,7,3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QChartView" name="originView">
       <property name="styleSheet">
        <string notr="true">background-color: #263238</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QChartView" name="catView">
       <property name="styleSheet">
        <string notr="true">background-color: #263238</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="9,2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>3</number>
       </property>
       <item>
        <widget class="QChartView" name="catedsView">
         <property name="styleSheet">
          <string notr="true">background-color: #263238</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QChartView" name="predsView">
         <property name="styleSheet">
          <string notr="true">background-color: #263238</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout" rowstretch="0,1,1,1,1,1">
       <item row="3" column="0">
        <widget class="QPushButton" name="pushButton_5">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Curlz MT</family>
           <pointsize>41</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	border: 1px solid rgb(173, 142, 180);
	border-radius:15px;
	color: rgb(159, 255, 64);
}

QPushButton:checked{
	background-color: rgb(175, 175, 175);
}
QPushButton:hover {
	background-color: rgb(74, 97, 109);
}

QPushButton:disabled{
	color: rgb(124, 124, 124);
};</string>
         </property>
         <property name="text">
          <string>5</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="pushButton_4">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Curlz MT</family>
           <pointsize>41</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	color:rgb(66, 255, 246);
	border: 1px solid rgb(173, 142, 180);
	border-radius:15px;
}

QPushButton:checked{
	background-color: rgb(175, 175, 175);
}
QPushButton:hover {
	background-color: rgb(74, 97, 109);
}
QPushButton:disabled{
	color: rgb(124, 124, 124);
};</string>
         </property>
         <property name="text">
          <string>4</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="pushButton_2">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Curlz MT</family>
           <pointsize>41</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	color: rgb(94, 56, 224);
	border: 1px solid rgb(173, 142, 180);
	border-radius:15px;
}

QPushButton:checked{
	background-color: rgb(175, 175, 175);
}
QPushButton:hover {
	background-color: rgb(74, 97, 109);
}

QPushButton:disabled{
	color: rgb(124, 124, 124);
};</string>
         </property>
         <property name="text">
          <string>2</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QPushButton" name="pushButton_del">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Agency FB</family>
           <pointsize>41</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	border: 1px solid rgb(173, 142, 180);
	border-radius:15px;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>C:/Users/fire-/Desktop/usertrashfull_92826.svg</normaloff>C:/Users/fire-/Desktop/usertrashfull_92826.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>56</width>
           <height>56</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="pushButton_3">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Curlz MT</family>
           <pointsize>41</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	color: rgb(222, 165, 55);
	border: 1px solid rgb(173, 142, 180);
	border-radius:15px;
}

QPushButton:checked{
	background-color: rgb(175, 175, 175);
}
QPushButton:hover {
	background-color: rgb(74, 97, 109);
}

QPushButton:disabled{
	color: rgb(124, 124, 124);
};</string>
         </property>
         <property name="text">
          <string>3</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QPushButton" name="pushButton_7">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Curlz MT</family>
           <pointsize>41</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	border: 1px solid rgb(173, 142, 180);
	border-radius:15px;
	color: rgb(69, 255, 120);
}

QPushButton:checked{
	background-color: rgb(175, 175, 175);
}
QPushButton:hover {
	background-color: rgb(74, 97, 109);
}
QPushButton:disabled{
	color: rgb(124, 124, 124);
};</string>
         </property>
         <property name="text">
          <string>7</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pushButton_1">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Curlz MT</family>
           <pointsize>41</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	color: rgb(244, 74, 77);
	border: 1px solid rgb(173, 142, 180);
	border-radius:15px;
}

QPushButton:checked{
	background-color: rgb(175, 175, 175);
}
QPushButton:hover {
	background-color: rgb(74, 97, 109);
}

QPushButton:disabled{
	
	color: rgb(124, 124, 124);
};</string>
         </property>
         <property name="text">
          <string>1</string>
         </property>
         <property name="shortcut">
          <string>F5</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <property name="autoRepeat">
          <bool>false</bool>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QPushButton" name="pushButton_9">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Curlz MT</family>
           <pointsize>41</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	color: rgb(222, 114, 58);
	border: 1px solid rgb(173, 142, 180);
	border-radius:15px;
}

QPushButton:checked{
	background-color: rgb(175, 175, 175);
}
QPushButton:hover {
	background-color: rgb(74, 97, 109);
}
QPushButton:disabled{
	color: rgb(124, 124, 124);
};</string>
         </property>
         <property name="text">
          <string>9</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="pushButton_stop">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Curlz MT</family>
           <pointsize>29</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	color: rgb(139, 69, 19);
	border: 1px solid rgb(173, 142, 180);
	border-radius:15px;
}

QPushButton:hover {
    color: #AFBDC4;
	background-color: rgb(74, 97, 109);
}
QPushButton:disabled{
	
	color: rgb(124, 124, 124);
};</string>
         </property>
         <property name="text">
          <string>STOP</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="pushButton_6">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Curlz MT</family>
           <pointsize>41</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	color: rgb(224, 76, 244);
	border: 1px solid rgb(173, 142, 180);
	border-radius:15px;
}

QPushButton:checked{
	background-color: rgb(175, 175, 175);
}
QPushButton:hover {
	background-color: rgb(74, 97, 109);
}

QPushButton:disabled{
	color: rgb(124, 124, 124);
};</string>
         </property>
         <property name="text">
          <string>6</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButton_start">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Chiller</family>
           <pointsize>29</pointsize>
           <italic>false</italic>
           <bold>false</bold>
           <strikeout>false</strikeout>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    color:rgb(247, 0, 124);
	border: 1px solid rgb(173, 142, 180);
	border-radius:15px;
}
QPushButton:hover {
    color: #AFBDC4;
	background-color: rgb(180, 148, 200);
}

QPushButton:pressed {
    color: #FFFFFF;
}

QPushButton:disabled{
	color: rgb(124, 124, 124);
};</string>
         </property>
         <property name="text">
          <string>START</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QPushButton" name="pushButton_8">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Curlz MT</family>
           <pointsize>41</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	border: 1px solid rgb(173, 142, 180);
	border-radius:15px;
	color: rgb(58, 127, 224);
}

QPushButton:checked{
	background-color: rgb(175, 175, 175);
}
QPushButton:hover {
	background-color: rgb(74, 97, 109);
}
QPushButton:disabled{
	color: rgb(124, 124, 124);
};</string>
         </property>
         <property name="text">
          <string>8</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget">
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QTableWidget {
    background-color: #263238;
    color: #546E7A;
    outline: 0;
    border: 2px solid transparent;
}

QTableWidget::item:hover {
    color: #AFBDC4;
    background: transparent;
}


QTableWidget::item:selected {
    color: #ffffff;
    background: transparent;
}</string>
     </property>
     <property name="rowCount">
      <number>9</number>
     </property>
     <property name="columnCount">
      <number>1</number>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <column/>
     <item row="0" column="0">
      <property name="text">
       <string/>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>PyQt6.QtCharts</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
